README.md

```diff
@@ -236,6 +238,22 @@ These can be set persistently via Maven settings (usually in `~/.m2/settings.xml
 </settings>%
 ```
 
+### S3 Tests
+
+To run `S3ClientTest` successfully, aws credentials and a set of pre-configured S3 buckets will be needed. There is a helper script from aws-c-s3 that can be used to set up the test environment, [here](https://github.com/awslabs/aws-c-s3/tree/main/tests/test_helper).
+
+To run the tests properly:
+
+``` sh
+cd aws-crt-java
+pip3 install boto3
```

Comment:
trivial: it's not installed as `pip3` on every platform. The most consistent way is:
```suggestion
python3 -m pip install boto3
```